homeassistant/components/intellifire/sensor.py

```diff
@@ -114,6 +114,12 @@ def _downtime_to_timestamp(data: IntellifirePollData) -> datetime | None:
         value_fn=lambda data: data.connection_quality,
         entity_registry_enabled_default=False,
     ),
+    IntellifireSensorEntityDescription(
+        key="errors",
+        name="Errors",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: ", ".join([x.name for x in data.error_codes]),
```

Comment:
Please add an .error_string property or similar to the PyPi lib so we don't have to mutate the data here